rusk-wallet/src/gql.rs

```diff
@@ -229,6 +268,33 @@ impl GraphQL {
         Ok(response)
     }
 
+    /// Query the archival node for moonlight transactions of `address`
+    /// in the given `block`.
+    ///
+    /// # Errors
+    /// This method errors if there was an error while sending the query,
+    /// or if the response body is not in JSON format or encoded correctly.
+    pub async fn moonlight_history_at_block(
+        &self,
+        address: &Address,
```

Comment:
```suggestion
        public_address: &Address,
```

Reply:
The `moonlight_history` function too uses `address` as its parameter name.
Should that be changed, too?